Lib/idlelib/idle_test/test_configdialog.py

```diff
@@ -44,7 +40,7 @@ def setUpModule():
 
 def tearDownModule():
     global root, configure
-    idleConf.userCfg = testcfg
+    idleConf.userCfg = usercfg
```

Comment:
I think you should rebase/merge `master` branch since this changed has been merged into via PR #2606.